main/templates/main/index.html

```diff
@@ -9,118 +9,97 @@
 
-			<div class="col-md-4" id="stat-3">
-				<div class="fa-stack">
-				  <i class="fa fa-circle fa-stack-2x"></i>
-				  <i class="fa fa-thumbs-up fa-stack-1x fa-inverse"></i>
-				</div>
+      <p>Curabitur at nisl mollis, sagittis elit et, molestie ligula. Maecenas sollicitudin ipsum aliquam, scelerisque turpis at, pretium mauris. Nulla dictum tincidunt quam, ut pulvinar turpis vehicula ultricies. Sed vestibulum convallis ex eu mattis. Maecenas congue posuere euismod. Sed id nulla enim. Curabitur vel lectus pellentesque, congue quam sed, ultricies nisi. Vestibulum id risus hendrerit, imperdiet enim vitae, ultrices est. Pellentesque faucibus ultricies est, id volutpat tellus tincidunt venenatis. Suspendisse vitae consectetur arcu, at laoreet lacus. Sed tristique posuere urna, ut iaculis metus vestibulum id. In laoreet neque ac diam posuere, ut sagittis nunc porta. Aliquam erat volutpat. Donec sit amet quam nec risus pharetra convallis.</p>
+    </div>
 
-				<div class="stat stat-type">Founded in</div>
-				<div class="stat stat-number">2014</div>
-			</div>
-		</div>
-	</div>
+  </div>
+  <a class="home-button" href="{% url 'our_team' %}">Our team</a>
 </section>
 
-<!--<section id="section-buttons">
-	<div class="container">
-			<a class="home-button" href="{% url 'projects' %}">View projects</a>
-			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-			<a class="home-button" href="{% url 'our_team' %}">View members</a>
-		
-	</div>
-</section>-->
-
-<section id="section-meeting">
-	<div class="container">
-		<h2>Our meetings</h2>
-		<div class="row">
-			<div class="col-md-6">
-				<div class="home-button">
-					<i class="fa fa-calendar fa-fw"></i>
-					Every Tuesday
-				</div>
-				<div class="home-button">
-					<i class="fa fa-clock-o fa-fw"></i>
-					6:00 PM - 6:30 PM
-				</div>
-
-				<div class="home-button">
-					<i class="fa fa-map-marker fa-fw"></i>
-					MC 4042
-				</div>
-			</div>
-			<div class="col-md-6">
-				<div class="embed-responsive embed-responsive-4by3">
-					<iframe class="embed-responsive-item" src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d723.8631673720687!2d-80.54398360716775!3d43.47202271829367!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x882bf6aa0a3f9659%3A0x713ea568814697ba!2sMathematics+and+Computer%2C+University+of+Waterloo%2C+200+University+Ave+W%2C+Waterloo%2C+ON+N2L+3G1!5e0!3m2!1sen!2sca!4v1424233496870" width="600" height="450" style="border:0"></iframe>
-				</div>
-			</div>
-		</div>
-	</div>
+<section id="section-projects">
+  <div class="container">
+    <h2>Our Projects</h2>
+    <div class="row">
+    {% for project in projects %}
+      <a href="{% url 'project_details' project.id %}">
```

Comment:
I think the <a> tag should be inside the column, instead of the other way around, so that if we have margins then the margins aren't also linked.